docs/src/examples.rst

```diff
@@ -65,7 +67,7 @@ gives you a leg up on using AWS services by providing the following benefits:
 * Automatically adds permissions for |ECR| if you use an image from |ECR|
   When you use an image from |ECR|, the |cdk| adds the correct permissions.
 
-* Convenient API for autoscaling
+* Automatic autoscaling
```

Comment:
What does this mean? It's not automatic yet. I think we may have miscommunicated

Reply:
Okay, so what should it say? Should I go back to:

* Convenient API for autoscaling
  The |cdk| supplies a method so you can autoscale instances when you use an |EC2| cluster;
  this functionality is done automatically when you use an instance in a Fargate cluster.